rackcity/utils/change_planner_utils.py

```diff
@@ -76,21 +79,41 @@ def detect_conflicts_cp(sender, **kwargs):
 
     # asset rack location conflicts with an active assetCP
     asset.location_conflict.clear()
-    for assetcp in AssetCP.objects.filter(
-        Q(rack=asset.rack_id)
-        & ~Q(related_asset=asset.id)
-        & Q(change_plan__execution_time=None)
-    ):
-        try:
-            validate_asset_location(
-                asset.rack_id,
-                assetcp.rack_position,
-                assetcp.model.height,
-                asset_id=assetcp.id,
-            )
-        except LocationException:
-            assetcp.asset_conflict_location = asset
-            AssetCP.objects.filter(id=assetcp.id).update(asset_conflict_location=asset)
+    if asset.model.is_rackmount():
+        for assetcp in AssetCP.objects.filter(
```

Comment:
would you mind renaming `assetcp` to `asset_cp`? it's more consistent with the rest of the code

Reply:
I didn't write this but will do

Reply:
I thought so, just giving u extra work 😘